eessi-2023.06-eb-4.8.0-2021a.yml

```diff
@@ -1,2 +1,3 @@
 easyconfigs:
   - GDAL-3.3.0-foss-2021a.eb
+  - ESPResSo-4.2.1-foss-2021a.eb
```

Comment:
@maxim-masterov `ERROR: One or more files not found: ESPResSo-4.2.1-foss-2021a.eb `

This easyconfig is not included yet with EasyBuild 4.8.0, so we need a `--from-pr` here, or use the just released EasyBuild v4.8.1 instead (the latter is the best option imho).

Reply:
my bad, thanks